src/dwn-http-server.ts

```diff
@@ -44,10 +44,11 @@ export class DwnHttpServer {
         try {
           dwnRequest = parseRequest(JSON.parse(req.headers['dwn-request'] as string))
           if (!dwnRequest.payload) {
-            const contentLength = req.headers['content-length']
+            const contentLength = req.headers['content-length'] ?? '0'
             const transferEncoding = req.headers['transfer-encoding']
             const requestDataStream = (parseInt(contentLength) > 0 || transferEncoding !== undefined) ? req : undefined
-            dwnRequest.payload = requestDataStream
+            if (requestDataStream)
+              dwnRequest.payload = await readReq(requestDataStream)
```

Comment:
was doing this in the `DwnProxy.listen()` function, but makes more sense semantically here